src/game/game.cpp

```diff
@@ -2418,6 +2418,9 @@ void Game::internalQuickLootCorpse(Player* player, Container* corpse) {
 		ss << "No loot";
 	}
 
+	if (player->checkAutoLoot()) {
+		ss << " (automatic looting)";
+	}
```

Comment:
> This might be a bit spammy to have as a message on every monster you kill, no?

I think this is not spam like a loot msg in every loot. It looks like an information suffix.